docs/docs/develop/structure.md

```diff
@@ -14,18 +14,21 @@ tags:
 
 {% band %}
 
+## Start the development server
+
 Run this command from the project root to start the build, watch, and server processes.
 ```bash
 npm run start
 ```
 
 The server will load on `http://localhost:8000` by default.
 
-<!-- TODO: Update Graphic to a more current image -->
-![npm run live-demo command](/images/develop/develop-structure.png)
-
+<!-- @TODO: Update Graphic to a more current image -->
 Assuming the `start` command started a server on port 8000, navigate to `http://localhost:8000/components/cool-element/demo/` to view your element.  Depending on other projects you are running via NPM the port number may differ.
```

Comment:
```suggestion
Assuming the `start` command started a server on port 8000, navigate to `http://localhost:8000/components/cool-element/demo/` to view your element.  Depending on other projects you are running via NPM the port number may differ.

The `start` command also starts an 11ty development server on `http://localhost:8080` by default.

Assuming the `start` command started a server on port 8080, navigate to `http://localhost:8080/components/cool-element/demo` to view your element.  Depending on other projects you are running via NPM the port number may differ.
```

Reply:
Nice suggestion.  Added and dropped in a screenshot to the 11ty docs section.